sdk/iOS/ZumoE2ETestApp/ZumoE2ETestApp/ZumoLoginTests.m

```diff
@@ -46,30 +46,36 @@ + (NSArray *)createTests {
     NSMutableArray *result = [[NSMutableArray alloc] init];
```

Comment:
Can you add a test for loginPrefix and alternate login host? Expected result can be 404